apps/client/src/app/core/scheme/indicators/indicators.component.ts

```diff
@@ -0,0 +1,277 @@
+    this.indexEdit = index
+    this.formGroup.controls.index.setValue(index)
+    this.formGroup.controls.name.setValue(name)
+    this.formGroup.controls.alias.setValue(alias)
+    this.formGroup.controls.helpText.setValue(helpText)
+  }
+
+  confirmationDeleteIndicator(event: Event, id: number, name: string) {
+    this.confirmationService.confirm({
+      target: event.target as EventTarget,
+      message: `¿Estás seguro de que quieres eliminar el indicador <strong>${name}</strong>?`,
+      header: 'Eliminar indicador',
+      icon: 'pi pi-info-circle',
+      acceptButtonStyleClass: 'p-button-danger p-button-text',
+      rejectButtonStyleClass: 'p-button-text p-button-text',
+      acceptIcon: 'none',
+      rejectIcon: 'none',
+      acceptLabel: 'Sí',
+      rejectLabel: 'No',
+
+      accept: () => {
+        this.toast.show('info', 'Eliminando..', 'Eliminando indicador..')
+        this.onDelete(id)
+      },
+      reject: () => {
+        this.toast.show('error', 'Rechazado', 'Haz rechazado la eliminación')
+      }
+    })
+  }
+  getAll() {
+    this.indicatorService.getAll(this.paginated).subscribe({
+      next: (res) => {
+        if (res.status === 'success') {
+          this.totalItems = res.data.totalItems
+          this.schemes = res.data.items.map((indicator) => {
+            return {
+              ...indicator,
+              categories: [],
+              criterias: []
+            }
+          })
+        }
+        this.isFetching = false
+        for (const scheme of this.schemes) {
+          this.getCategoriesPerIndicator(scheme.index)
+          this.getCriterionPerIndicator(scheme.index)
+        }
+      },
+      error: (e) => {
+        console.error(e)
+        this.toast.show('error', 'Error', e.error.data.message)
+        this.isFetching = false
+      }
+    })
+  }
+
+  getCategoriesPerIndicator(indicatorId: number) {
+    this.categoryService
+      .getAllCategoriesByIndicatorIndex(indicatorId)
+      .subscribe({
+        next: (res) => {
+          if (res.status === 'success') {
+            this.schemes.map((scheme) => {
+              if (scheme.index === indicatorId) {
+                scheme.categories = res.data
+              }
+            })
+          }
+        },
+        error: (e) => {
+          console.error(e)
+          this.toast.show('error', 'Error', e.error.data.message)
+        }
+      })
+  }
+
+  getCriterionPerIndicator(indicatorId: number) {
+    this.criteriaService
+      .getAllCriterionByIndicatorIndex(indicatorId)
+      .subscribe({
+        next: (res) => {
+          if (res.status === 'success') {
+            this.schemes.map((scheme) => {
+              if (scheme.index === indicatorId) {
+                scheme.criterias = res.data
+              }
+            })
+          }
+        },
+        error: (e) => {
+          console.error(e)
+          this.toast.show('error', 'Error', e.error.data.message)
+        }
+      })
+  }
+
+  onCreate() {
```

Comment:
No estás chequeando que el form sea válido

```suggestion
  onCreate() {
    if (this.formGroup.invalid) return
    ...
```